src/boot/options.nix

```diff
@@ -0,0 +1,329 @@
+iogx-inputs:
+
+let
+
+  l = builtins // iogx-inputs.nixpkgs.lib;
+
+
+  default-pre-commit-hook = {
+    enable = false;
+    extraOptions = "";
```

Comment:
Why is this needed?

Reply:
We want to give the user the option to pass additional options to the formatters.
For example one repository want to pass the `--tabulate` flag to `cabal-fmt`